odoo_sh/getting_started/branches.rst

```diff
@@ -233,6 +235,35 @@ The *import database* feature accepts database archives in the format provided b
 * the Odoo.sh backup download button of this *Backups* tab,
 * the Odoo.sh dump download button in the :ref:`Builds view <odoosh-gettingstarted-builds>`.
 
+Settings
+--------
+Here you can find a couple of settings that only apply to the currently selected branch.
+
+.. image:: ./media/interface-branches-settings.png
+   :align: center
+
+**Mute Branch**
+
+You have the possibility to mute a branch, so that new commits won't trigger builds on Odoo.sh. You can activate this
+should you have branches that are used for other purposes than on Odoo.sh. By default, when creating a project from an
```

Comment:
```
You can activate this should you have branches that are used for other purposes than on Odoo.sh.
```
Despite the fact this is most-likely correct English considering your background, I  had to re-read this multiple times to understand it : D. Maybe we can simplify this for dummy people like me ! Or don't if you believe this is perfectly clear